scripts/deployment/start_containers.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+## CLI ARGUMENTS ---
+
+# default argument values
+WEBSITE_ARG=""
+INTERACTIVE=false
+CLEAN=false
+RESET=false
+
+# function to display usage help
+usage() {
+    echo "Usage: $0 [--website] [--interactive] [--clean] [--reset]"
+    echo "  --website      If set, runs the website container"
+    echo "  --interactive  If set, run commands inside the sailbot workspace container interactively"
+    echo "  --clean        If set, removes containers after running"
+    echo "  --reset        If set, removes containers and volumes after running"
+    exit 1
+}
+
+# parse command-line options
+while getopts ":-:" opt; do
+    case "${opt}" in
+        -)
+            case "${OPTARG}" in
+                website)
+                    WEBSITE_ARG="--file .devcontainer/website/docker-compose.website.yml"
```

Comment:
@JordanChen123 do you think it makes more sense to run the dev or prod version of the website on the raspberry pi for testing purposes?

Reply:
For testing purposes on the raspberry pi, it would make more sense to run the production version, since the production build is optimized for performance and can provide us a more accurate representation of how the website will behave in practice, but if we are expecting to do a lot of detailed debugging and/or make changes frequently while testing, it would be better to run the dev version so we don't have to rebuild the entire application every time we make a change. 

In summary, probably would be best to run the prod version, unless we are frequently having to make changes while testing then we should switch to the dev version.

Reply:
I tried to use the prod version, but I am getting this error after updating this branch with main

![image](https://github.com/UBCSailbot/sailbot_workspace/assets/65756895/c9f4484e-d2de-4149-829f-c6108dd64a39)

Reply:
Pull the recent changes from main and try again, it should work now

Reply:
It works now, thanks. However, the prod version takes almost 30 seconds to start (I think most of it is spent generating static pages). Is there any way of speeding this up?

Reply:
Interesting, I do have a few ideas that could speed up the load time, but I am not sure how impactful it could be. I will look into it more

Reply:
I'll switch it to the prod version then, but if there's issues that pop up because of the slow startup time you can switch it back

https://github.com/UBCSailbot/sailbot_workspace/pull/396/commits/77e15d51eddeb07bf02afa3a1cb418749830fd09